lib/core/plugins/context.js

```diff
@@ -409,4 +408,16 @@ function curryRemoveStrategy(kuzzle, pluginName) {
   };
 }
 
+/**
+ * Adds the plugin name to log lines
+ *
+ * @param {String} pluginName
+ * @param {String[]} lines
+
+ * @returns {String[]}
+ */
+function addPluginName (pluginName, lines = []) {
```

Comment:
I'm not sure about what happens when you send multiple arguments to log functions. 
If `console` is invoked (before Kuzzle logger has started), then those aren't lines. And if this is winston (99% of the time), I think other arguments after the 1st one are ignored (to be verified).

Either way, I don't think you have to worry about multiple arguments, especially considering that our documentation says nothing of the sort: https://docs.kuzzle.io/core/2/plugins/plugin-context/log/error/

This means that you could simplify the code by replacing this:
`warn: (...args) => kuzzle.log.warn(...addPluginName(pluginName, args))`

with that:
`warn: message => kuzzle.log.warn(`[${pluginName}] - ${message}`)`